zhaquirks/xiaomi/aqara/switch_agl011.py

```diff
@@ -0,0 +1,153 @@
+"""Quirk for Aqara Dimmer Switch H2 EU (lumi.switch.agl011)."""
+
+from typing import Any
+
+from zigpy import types
+from zigpy.profiles import zha
+from zigpy.quirks.v2 import QuirkBuilder
+from zigpy.zcl.foundation import BaseAttributeDefs, ZCLAttributeDef
+
+from zhaquirks.xiaomi import DeviceTemperatureCluster, XiaomiCluster
+
+
+class OperationMode(types.enum8):
+    """Enum for dimmer operation mode."""
+
+    Decoupled = 0x00
+    Relay = 0x01
+
+
+class Phase(types.enum8):
+    """Enum for dimmer phase."""
+
+    Leading = 0x00
+    Trailing = 0x01
+
+
+class PowerOnBehaviour(types.enum8):
+    """Enum for dimmer power-on behaviour."""
+
+    On = 0x00
+    Previous = 0x01
+    Off = 0x02
+    Inverted = 0x03
+
+
+class OppleCluster(XiaomiCluster):
+    """Aqara manufacturer-specific cluster for the dimmer switch H2 EU."""
+
+    cluster_id = 0xFCC0
+    ep_attribute = "opple_cluster"
```

Comment:
Please inherit from `XiaomiAqaraE1Cluster` here. It already sets the `cluster_id` and `ep_attribute`.
(Yes, I'm aware the name isn't ideal.)

```suggestion
class OppleCluster(XiaomiAqaraE1Cluster):
    """Aqara manufacturer-specific cluster for the dimmer switch H2 EU."""
```